### ISSUE TITLE
Create dataset loader for mySentence

### ISSUE THREAD

**Issue by SamuelCahyawijaya** [SEACrowd/seacrowd-datahub#264]
Dataloader name: `mysentence/mysentence.py`
DataCatalogue: http://seacrowd.github.io/seacrowd-catalogue/card.html?mysentence

|     Dataset| mysentence |
|-------------|---|
| Description | mySentence is a corpus with a total size of around 55K for Myanmar sentence segmentation.  In formal Burmese (Myanmar language), sentences are grammatically structured and typically end with the "။" pote-ma symbol. However, informal language, more commonly used in daily conversations due to its natural flow, does not always follow predefined rules for ending sentences, making it challenging for machines to identify sentence boundaries. In this corpus, each token of the sentences and paragraphs is tagged from start to finish. |
| Subsets | sentences, sentences+paragraphs |
| Languages | mya |
| Tasks | POS Tagging |
| License | Creative Commons Attribution Non Commercial Share Alike 4.0 (cc-by-nc-sa-4.0) |
| Homepage | https://github.com/ye-kyaw-thu/mySentence |
| HF URL | - |
| Paper URL | https://ph05.tci-thaijo.org/index.php/JIIST/article/view/87 |

**Comment by Gyyz** [comment]
#self-assign

**Comment by sabilmakbar** [comment]
Hi @SamuelCahyawijaya, the paper contained in dataloader card is different from the one that I checked on the Data Repo.

The Github repo points to [this paper](https://link.springer.com/chapter/10.1007/978-3-031-36886-8_24) which was published in 2023 (and the initial commit on the repo is on Dec 2022), whereas the paper in the datacard was released in 2017 (way earlier than both Github Repo and its paper).

Should we add both papers as the citation info, or keep either one of these?
